### ISSUE TITLE
env: node\r: No such file or directory

### ISSUE THREAD

**Issue by Aetherus** [leftstick/Sero-cli#1]
On MAC, when try to boot the server, it throws `env: node\r: No such file or directory`. After converting all CR to CRLF, this error is solved.

**Comment by leftstick** [comment]
Which file has to be converted? The file in sero-cli, or in your web ?

**Comment by Aetherus** [comment]
In sero-cli. I executed the following command:

``` bash
find ~/.nvm/versions/io.js/v2.5.0/lib/node_modules/sero-cli -name "*.js" | xargs dos2unix
```

then all is fine.

That command fixed quite a few files.

**Comment by leftstick** [comment]
This is really weird, i just checked it with my mac, the files contains `crlf` were dependencies used in `sero-cli`, they are not provided by me.

But using `dos2unix xxx/lib/node_modules/sero-cli/bin/Sero.js`, all problems gone. No idea what character contains in `Sero.js` 
But still interesting, i will look into it

**Comment by leftstick** [comment]
Fixed in `2.4.10`
